pontoon/base/migrations/0040_locale_translators.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models, migrations
+
+ACTIVITY_THRESHOLD = 10
+"""Minimal number of approved translations to promote contributor into translator."""
+
+
+def create_translators(apps, schema_editor):
+    """
+    Bug 952488 - We're assigning translation permissions to active contributors.
+    """   
+    User = apps.get_model('auth', 'User')
+
+    # We're using through migration because it's easier to map
+    # user/group primary keys without requiring to query full objects.
+    UserGroup = User.groups.through
+
+    contributors_locale = (
+        User.objects.exclude(email=settings.EXCLUDE)
+            .filter(translation__approved=True)
+            .values('pk', 'translation__locale')
+            .annotate(approved_translations=models.Count('translation__locale', distinct=True))
+            .filter(approved_translations__gte=ACTIVITY_THRESHOLD)
```

Comment:
Instead, you should filter users, that have only submitted translations to one locale. We'll take care of the others manually.